index.bs

```diff
@@ -108,8 +108,32 @@ Every {{Document}} has a <dfn for="Document">prerendering browsing contexts map<
     1. [=Queue a global task=] on the [=networking task source=] given |successorBC|'s [=browsing context/active window=] to perform the following steps
 
       1. Set |successorBC|'s [=browsing context/loading mode=] to "`default`".
+
+      1. Fire an event named `prerenderingchange`, with its {{Event/cancelable}} and {{Event/bubbles}} attributes initialized to false, at |successorBC|'s [=active document=].
 </div>
 
+Amend the {{Document}} interface:
+
+<pre class='idl'>
+  partial interface Document {
+      readonly attribute boolean prerendering;
+      attribute EventHandler onprerenderingchange;
+  };
+</pre>
+
+<dl dfn-type=attribute dfn-for=Document>
+  <dt>
+    <dfn>prerendering</dfn>
+    <dd>Is true if [=Document=] has a non-null [=Document/Browsing Context=] and its loading mode is "`prerender`" or "`uncredentialed-prerender`", false otherwise.</dd>
+    <p class="note">This attribute lets pages know when they're being presented in a non-interactive "prerendering-like" context. In the future, this would include a visible document in a `<portal>` element, both when loaded into it or via predecessor adoption.</p>
+  </dt>
+
+  <dt>
+    <dfn>onprerenderingchange</dfn>
+    <dd>This event handler, of event handler type `prerenderingchange`, MUST be fired whenever the value of the {{Document/prerendering}} attribute changes.</dd>
```

Comment:
This is not an appropriate place to put MUST requirements, especially one that overlaps (contradicts?) the above event firing. Instead this should just be defined like other IDL event handlers, e.g. https://html.spec.whatwg.org/#audiotracklist-and-videotracklist-objects:event-handler-idl-attributes:

> The `onprerenderingchange` attribute is an [=event handler IDL attribute=] corresponding to the `prerenderingchange` [=event handler event type=].

Reply:
Done.